docs/installguide/uninstall.rst

```diff
@@ -11,14 +11,22 @@ _______
 Mac OSX
 _______
 
-1. Launch ``KA-Lite Monitor`` from your ``Applications`` folder.
+Option 1:
```

Comment:
Call it:

    Uninstallation from user interface
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~